lib/rmt/scc.rb

```diff
@@ -84,7 +84,7 @@ def sync_systems
     scc_api_client = SUSE::Connect::Api.new(Settings.scc.username, Settings.scc.password)
 
     # do not sync BYOS proxy systems to SCC
-    systems = System.where('scc_registered_at IS NULL OR last_seen_at > scc_registered_at').where(proxy_byos: false)
+    systems = System.where('scc_registered_at IS NULL OR last_seen_at > scc_registered_at').not_byos
```

Comment:
Wouldn't this include `:not_applicable` ?

Reply:
yes, same behavior as before